src/unity/python/setup.py

```diff
@@ -147,24 +147,154 @@ def run(self):
                      '*.dll', '*.def',
                      'deploy/*.jar', '*.exe', 'libminipsutil.*',
                      'mxnet/*.ttf',
-                    'Turi Create Visualization.app/Contents/_CodeSignature/CodeResources',
-                    'Turi Create Visualization.app/Contents/Frameworks/*.dylib',
-                    'Turi Create Visualization.app/Contents/Info.plist',
-                    'Turi Create Visualization.app/Contents/Resources/*.car',
-                    'Turi Create Visualization.app/Contents/Resources/*.css',
-                    'Turi Create Visualization.app/Contents/Resources/*.icns',
-                    'Turi Create Visualization.app/Contents/Resources/*.js',
-                    'Turi Create Visualization.app/Contents/Resources/*.html',
-                    'Turi Create Visualization.app/Contents/Resources/Base.lproj/Main.storyboardc/XfG-lQ-9wD-view-m2S-Jp-Qdl.nib',
-                    'Turi Create Visualization.app/Contents/Resources/Base.lproj/Main.storyboardc/NSViewController-99M-uP-3Iu.nib',
-                    'Turi Create Visualization.app/Contents/Resources/Base.lproj/Main.storyboardc/mainWindow.nib',
-                    'Turi Create Visualization.app/Contents/Resources/Base.lproj/Main.storyboardc/MainMenu.nib',
-                    'Turi Create Visualization.app/Contents/Resources/Base.lproj/Main.storyboardc/Info.plist',
-                    'Turi Create Visualization.app/Contents/Resources/Base.lproj/Main.storyboardc/mainWindowRun.nib',
-                    'Turi Create Visualization.app/Contents/Resources/Base.lproj/Main.storyboardc/99M-uP-3Iu-view-SkP-0p-uFQ.nib',
-                    'Turi Create Visualization.app/Contents/Resources/*.dylib',
-                    'Turi Create Visualization.app/Contents/PkgInfo',
-                    'Turi Create Visualization.app/Contents/MacOS/Turi Create Visualization'
+
+                     # macOS visualization
+                     'Turi Create Visualization.app/Contents/_CodeSignature/CodeResources',
+                     'Turi Create Visualization.app/Contents/Frameworks/*.dylib',
+                     'Turi Create Visualization.app/Contents/Info.plist',
+                     'Turi Create Visualization.app/Contents/Resources/*.car',
+                     'Turi Create Visualization.app/Contents/Resources/*.css',
+                     'Turi Create Visualization.app/Contents/Resources/*.icns',
+                     'Turi Create Visualization.app/Contents/Resources/*.js',
+                     'Turi Create Visualization.app/Contents/Resources/*.html',
+                     'Turi Create Visualization.app/Contents/Resources/Base.lproj/Main.storyboardc/XfG-lQ-9wD-view-m2S-Jp-Qdl.nib',
```

Comment:
Could we just do 'Turi Create Visualization.app/Contents/Resources/Base.lproj/Main.storyboardc/*.nib' ?